backend/src/main/java/c/team/account/UserAccountRepository.java

```diff
@@ -6,7 +6,6 @@
 import java.util.UUID;
 
 public interface UserAccountRepository extends MongoRepository<UserAccount, UUID> {
-
     UserAccount findAccountByUsername(String username);
-
+    UserAccount findAccountById(String id);
```

Comment:
Chyba MongoRepository ma wbudowaną funkcję `findById(String id)`, więc to może nie być potrzebne

Reply:
Ma findById(UUID id) i przez to pisałem własną :P

Reply:
A dobra, bo jak zmienisz UUID na String w "szablonie" MongoRepository to faktycznie XD